monitor.c

```diff
@@ -147,14 +142,16 @@ int read_dev_state(int fd)
 	while (cp) {
 		if (sysfs_attr_match(cp, "faulty"))
 			rv |= DS_FAULTY;
-		if (sysfs_attr_match(cp, "in_sync"))
+		else if (sysfs_attr_match(cp, "in_sync"))
 			rv |= DS_INSYNC;
-		if (sysfs_attr_match(cp, "write_mostly"))
+		else if (sysfs_attr_match(cp, "write_mostly"))
 			rv |= DS_WRITE_MOSTLY;
-		if (sysfs_attr_match(cp, "spare"))
+		else if (sysfs_attr_match(cp, "spare"))
 			rv |= DS_SPARE;
-		if (sysfs_attr_match(cp, "blocked"))
+		else if (sysfs_attr_match(cp, "blocked"))
 			rv |= DS_BLOCKED;
+		else if (sysfs_attr_match(cp, "external_bbl"))
+			rv |= DS_EXTERNAL_BB;
```

Comment:
This could use maps if you'd like to make it extra legit, but I'm fine with this as it is.

Reply:
This should go separately because as you probably know, there will be more places like that :(
I think that I will revert this change to not mess with not important rework here.

Reply:
And in fact, it was bad change.